pkg/kademlia/export_test.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package kademlia
+
+var MaxBins = maxBins
+var TimeToRetry = &timeToRetry
```

Comment:
Why this value has to be a pointer?

Reply:
because i'm overriding the value in `TestBackoff`